### DISCUSSION TITLE
为什么APP界面这里一直是未登录啊

### DISCUSSION THREAD

**Discussion by Fluorine-z** [share121/inter-knot#5141] [灌水]
如题，一直显示未登录但是我已经在发帖了
![Uploading Screenshot_2024-09-18-08-59-40-382_com.share121.inter_knot.jpg…]()

**Comment by Fluorine-z** [comment]
![Screenshot_2024-09-18-08-59-40-382_com share121 inter_knot](https://github.com/user-attachments/assets/b7f61074-ff12-43b4-b698-f7432224ff63)
补个图

**Comment by March-777th** [comment]
在 `我的` 界面里选择用户接口就能登陆了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
![Screenshot_2024-09-18-09-38-05-96_219376f1dbde3085d7f592166ad0f554](https://github.com/user-attachments/assets/ee2bdb7f-b950-4a59-acee-5f678ba8b418)

**Comment by Fluorine-z** [comment]
搞好了，感谢
